### PR TITLE
Add example of colouring branches in a focal tree

### PR DESCRIPTION
**Pull request by hyanwong**
As discussed with @petrelharp on slack

### CODE REVIEW

File: viz.md [hunk 1]
```diff
@@ -455,14 +469,50 @@ the following element:
 <g class="a10 i3 leaf m16 m17 node n7 p2 s15 s16 sample">...</g>
 ```
 
-This makes it easy to e.g. colour branches that are shared between trees (i.e. that
+#### Styling graphical elements 
+
+The classes above make it easy to target specific nodes or edges in one or multiple
+trees. For example, we can colour branches that are shared between trees (i.e. that
 have the same parent and child):
```
Comment (petrelharp) [inline viz.md:476]:
```suggestion
trees. For example, we can colour branches that are shared between trees
(identified here as ones that have the same parent and child):
```

File: viz.md [hunk 2]
```diff
@@ -455,14 +469,50 @@ the following element:
...
 SVG(ts_small.draw_svg(time_scale="rank", size=wide_fmt, style=css_string))
 ```
 
+By generating the css string programatically, you can target all the edges present in a
+particular tree, and see how they gradually disappear from adjacent trees. Here, for
+example the branches in the central tree have been coloured red, as have the identical
+branches in adjacent trees. The central tree represents a location in the genome
+that has seen a selective sweep, and therefore has short branch lengths: adjacent trees
+are not under direct selection and thus the black branches tend to be longer. For
+visual clarity, node symbols and labels have been turned off.
```
Comment (petrelharp) [inline viz.md:489]:
```suggestion
are not under direct selection and thus the black branches tend to be longer.
These (red) shared branches extending far on either side represent shared haplotypes,
and this shows how long, shared haplotypes can extend much further away from a sweep
than the region of reduced diversity (which is the region spanned by the short tree in the middle).
For visual clarity, node symbols and labels have been turned off.
```

**Comment by petrelharp** [comment]
I made some suggestions. This is excellent!

**Comment by hyanwong** [comment]
All LGTM. For some reason I'm getting "Invalid email for web commit." when trying to accept your changes, so if you can accept them for me and merge, just go ahead.

**Comment by petrelharp** [comment]
> The other possibility is to put this in the "examples" section at the end of the viz tutorial.

Whoops - just saw this after I merged. As you see fit.

**Comment by petrelharp** [comment]
Hm, yes - that's a good idea, though.

**Comment by hyanwong** [comment]
It's done now! Let's leave it for the moment: I will have to change the ARG viz examples anyway, so could move it at the same time.

**Pull request merged**